modules/rust.sh

```
@@ -16,7 +19,14 @@ export PATH="$HOME/.cargo/bin:$PATH"
 rustup component add clippy
 rustup component add rustfmt
 
-cargo install cargo-edit # upgrade dependencies from the CLI
-cargo install cargo-make # useful task runner
+# add 'cargo-binstall' to speed up later installs.
+cargo_binstall_tmp="$install_tmp_dir/cargo-binstall-installer.sh"
```

Shellcheck (0.11.0)

[warning] install_tmp_dir is referenced but not assigned.

(SC2154)